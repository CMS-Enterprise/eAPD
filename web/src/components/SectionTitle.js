import PropTypes from 'prop-types';
import React from 'react';

const SectionTitle = ({ children }) => (
  <h2 className="mt1 mb2 h1 sm-h0 teal light">{children}</h2>
);

SectionTitle.propTypes = {
  children: PropTypes.node.isRequired
};

export default SectionTitle;
