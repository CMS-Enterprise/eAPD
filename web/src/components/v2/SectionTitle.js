import PropTypes from 'prop-types';
import React from 'react';

const SectionTitle = ({ children }) => (
  <h2 className="mt1 mb2 fw-800">{children}</h2>
);

SectionTitle.propTypes = {
  children: PropTypes.node.isRequired
};

export default SectionTitle;
