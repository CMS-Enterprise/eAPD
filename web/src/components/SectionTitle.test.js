import { shallow } from 'enzyme';
import React from 'react';

import SectionTitle from './SectionTitle';

describe('SectionTitle component', () => {
  test('renders correctly', () => {
    const component = shallow(
      <SectionTitle>
        <div className="test child" />
      </SectionTitle>
    );
    expect(component).toMatchSnapshot();
  });
});
