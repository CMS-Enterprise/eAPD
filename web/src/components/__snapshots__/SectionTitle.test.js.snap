// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SectionTitle component renders correctly 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SectionTitle>
    <div
      className="test child"
    />
  </SectionTitle>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="test child"
      />,
      "className": "mt1 mb2 fw-800",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "className": "test child",
      },
      "ref": null,
      "rendered": null,
      "type": "div",
    },
    "type": "h2",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="test child"
        />,
        "className": "mt1 mb2 fw-800",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "className": "test child",
        },
        "ref": null,
        "rendered": null,
        "type": "div",
      },
      "type": "h2",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
